### PR TITLE
Update redirect-incoming-requests.md

### LINKED ISSUE #972: Redirect Incoming Requests Doc Update
Re: [Redirect Incoming Requests](https://pantheon.io/docs/articles/sites/code/redirect-incoming-requests)
## Issue Description:
The redirect to www is superfluous with the dub-dub-dubber IP address recommended for Bare domains.
Removing www code block should be accompanied with a link to the serve site from bare domain anchor on the `sites/domains` doc. It also causes a redirect loop when used in conjunction with the a-record to the dubdubdubber
## Suggested Resolution

### PR DESCRIPTION
**Pull request by ttrowell**
Closes #972
You can't use the second snippet to standardize on a bare domain without a CNAME flattening service.  So, we need to add the steps to set that up.  @timani Please review for accuracy.  Thanks!

### CODE REVIEW

**Comment by ttrowell** [comment]
I don't think I have the spacing right on the 'Note' part @rachelwhitton .  help?

**Comment by rachelwhitton** [comment]
Thanks for the PR @ttrowell! Made some minor adjustments, but the spacing in the Note div looked good to me! 

+1

@nataliejeremy can you review?

File: source/docs/articles/sites/code/redirect-incoming-requests.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ Automatic resolution of domains is not supported. For each domain that you want
 
 ## Redirect to a Common Domain
 
-While it’s good for visitors and DNS to resolve both www and the domain itself, it's best practice to choose one or the other and redirect from www to non-www (or vice versa, your call). This optimizes SEO by avoiding duplicate content and prevents session strangeness, where a user can be signed on one domain but logged out of other domains at the same time.  Choose one of the blocks of code below:
+While it’s good for visitors and DNS to resolve both www and the domain itself, it's best practice to choose one or the other and redirect from www to non-www (or vice versa, your call). This optimizes SEO by avoiding duplicate content and prevents session strangeness, where a user can be signed on one domain but logged out of other domains at the same time.  Choose the following block to standardize on the www domain:
 ```
-    // Require WWW.
+    // Redirect all traffic to WWW. For example: www.yoursite.com
```
Comment (bmackinney) [inline source/docs/articles/sites/code/redirect-incoming-requests.md]: Isn't this only necessary for `https` traffic? the dub-dub-dubber does this already.

Comment (bmackinney) [inline source/docs/articles/sites/code/redirect-incoming-requests.md]: see: https://github.com/pantheon-systems/documentation/issues/972

**Comment by nataliejeremy** [comment]
@ttrowell Can you take a look at Brian's question? Thanks.

**Comment by rachelwhitton** [comment]
@ttrowell @bmackinney can you review and +1 changes?

Description edited to close related issue.

**Comment by rachelwhitton** [comment]
@bmackinney updated, thanks for the review! I think we're ready to :ship:

File: source/docs/articles/sites/code/redirect-incoming-requests.md [hunk 2]
```diff
@@ -30,21 +30,39 @@ Automatic resolution of domains is not supported. For each domain that you want
...
+
+
+
+If you prefer to use the bare domain, use the following code block and run your DNS settings through a service that supports CNAME flattening, such as CloudFlare. For details, see [Cloudflare CDN and DNS on Pantheon](/docs/articles/sites/cloudflare-cdn-and-dns).
+
+<div class="alert alert-info" role="alert">
+<h4>Note</h4>
+If you are running the site on a Pro plan or above with an SSL certificate, use the snippet below without configuring CloudFlare. </div>
+
+To direct all traffic to the bare domain using CloudFlare:
```
Comment (bmackinney) [inline source/docs/articles/sites/code/redirect-incoming-requests.md]: not "using CloudFlare." -> "you will need to use a domain name host that offers CNAME flattening. See (our doc) section (in `/sites/domains`) and.. add the following..."

**Comment by rachelwhitton** [comment]
ping @bmackinney - can you review and merge?

**Comment by bmackinney** [comment]
+1

**Pull request merged**